src/main.cpp

```diff
@@ -385,26 +385,6 @@ void vcMain_MainLoop(vcState *pProgramState)
         }
 #endif //VC_HASCONVERT
 
-        // test to see if specified filepath is valid
-        udFile *pTestFile = nullptr;
-        udResult result = udFile_Open(&pTestFile, pNextLoad, udFOF_Read);
-        if (result == udR_Success)
-        {
-          udFile_Close(&pTestFile);
-        }
-        else
-        {
-          vcState::ErrorItem status;
-          status.source = vcES_File;
-          status.pData = pNextLoad; // this takes ownership so we don't need to dup or free
-          status.resultCode = result;
-
-          pNextLoad = nullptr;
-
-          pProgramState->errorItems.PushBack(status);
```

Comment:
Without this code block, do load failures still end up in the error window?

Reply:
Only as scene item errors